packages/graphql/src/schemaTypes/objectTypes/gql-Mutation.ts

```diff
@@ -585,11 +585,7 @@ export const mutation = mutationType({
         projectId: nonNull(stringArg()),
       },
       resolve: async (_, args, ctx) => {
-        try {
-          await ctx.actions.project.setProjectIdInConfigFile(args.projectId)
-        } catch {
-          // ignore error as not useful for end user to see
```

Comment:
🤣 😭 this ignored error was painful for me to track down. I think there's value in having it:

Even if it's not immediately useful for the end user to see, it becomes very useful if/when the end user reports a bug. Without this error, the "Create project" button just fails silently.